src/SFA.DAS.Commitments.Database/Tables/Commitment.sql

```diff
@@ -7,6 +7,7 @@
     [LegalEntityName] NVARCHAR(100) NOT NULL, 
     [ProviderId] BIGINT NULL,
     [ProviderName] NVARCHAR(100) NULL,
-	[CommitmentStatus] SMALLINT NOT NULL DEFAULT 0, 
-    [EditStatus] SMALLINT NOT NULL DEFAULT 0 
+    [CommitmentStatus] SMALLINT NOT NULL DEFAULT 0, 
+    [EditStatus] SMALLINT NOT NULL DEFAULT 0,
+    [CreatedOn] DATETIME NULL
```

Comment:
Same as Apprenticeship. I think this should be not null. Might be a pain when we deploy.